client/src/components/UI/NavBar.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+
+
+const styles = {
+  root: {
+    flexGrow: 1,
+  },
+  grow: {
+    flexGrow: 1,
+  }
+};
+
+function ButtonAppBar({ classes, isLoggedIn }) {
+  return (
```

Comment:
As ButtonAppBar is just returning JSX you can directly return ie:
Use this signature ButtonAppBar = () => (<JSX />)
Check App.jsx for reference.